private/jsx/pixels.js

```diff
@@ -6,6 +6,7 @@ export default class Pixels {
     // dataArray will be an array which holds the x, y, and value for each object
     // Example: Float32Array([x1, y1, v1, x2, y2, v2, ...])
     constructor(
+        name,
```

Comment:
use a hashed 'key' is better than use the 'name' i think.

Reply:
maybe like this one.
https://github.com/cesandoval/PaintingWithData_Riyadh/blob/26f5b40f75605540c7a87a2055a0284163635fd6/private/jsx/vprog/rVpl.js#L1089-L1092